### DISCUSSION TITLE
Bug for 3.2

### DISCUSSION THREAD

**Discussion by RaymondFishWang** [apache/dolphinscheduler#15261] [General]
one task quote the file from resource center , if I change the file name , the orignial file is invisible in the task , but  the file information exists in database, 
Then I start the task. DS will issue the error information: the file doesn't exists.  And I also can't delete the file from the task.

**Comment by zui01** [comment]
task中记录的是资源id，不是资源名。资源中心重新上传会改id，哪怕名字一样，就会找不到

**Reply by RaymondFishWang** [reply]
you can check the table t_ds_task_instance-column task_params, it will record the full file path and filename
